core/tests/webdriverio_utils/workflow.js

```diff
@@ -416,6 +416,29 @@ var createQuestion = async function() {
   await skillEditorPage.saveQuestion();
 };
 
+var changeQuestionInteraction = async function() {
```

Comment:
I think this needs to be more specific about how it changes the question. From reading the name of the function I don't think it's a clear description of all the details that the function is doing.

If you can't find a good overarching name that properly describes what the function is doing, then you might need multiple functions that each do a specific easily-describable thing. Those functions might also need to be parameterized.

Reply:
Seems good, please add a comment at the top though explaining what this function does exactly.